dbt/include/databricks/macros/adapters.sql

```diff
@@ -139,3 +87,30 @@
     {% endfor %}
   {% endif %}
 {% endmacro %}
+
+
+{# backward compatitibily #}
+{% macro dbt_databricks_file_format_clause() %}
```

Comment:
I wonder if it's possible to add a test for this compatibility.

Reply:
The main macros such as `databricks__create_table_as` are still using the old name macros and there is a unit test `tests/unit/test_macros.py`.

Reply:
> The old name macros are still available during 1.0.x releases, but will not be available in 1.1.0 release.

This approach makes sense to me!